package.json

```diff
@@ -12,12 +12,18 @@
     "url": "https://bitbucket.org/cnndatastore/node-inews.git"
   },
   "main": "inewsFtp.js",
+  "scripts": {
+    "test": "jest"
+  },
   "dependencies": {
     "camelcase": "3.0.x",
     "ftp": "0.3.x",
     "htmlparser": "1.7.x",
+    "jobs-queue": "0.0.x",
     "unescape": "^0.2.0",
-    "wolfy87-eventemitter": "5.1.x",
-    "jobs-queue": "0.0.x"
+    "wolfy87-eventemitter": "5.1.x"
```

Comment:
Is this library necessary? Wouldn't either the EventEmitter or EventTarget api be sufficient?